fares-data-build-tool/fdbt-site
# Feature/tfn 472 enable checkbox opt in

# Description

-   To make it so that the option the user selects for the checkbox on if they want to be contactable is listened to and recorded in cognito. Also updated FDBT-AWS repo for cognito CF changes.

# Testing instructions

-   Run MFA script, and then the fdbt-dev make command to run the site. Send yourself an invitation through cognito to the site, use the email but alter the URL to include localhost. Then register and check the checkbox. Once complete, go to cognito and ensure that your user has 'contactable: yes'.

# Type of change


# Checklist:
